### DISCUSSION TITLE
디자인 관련 논의

### DISCUSSION THREAD

**Discussion by 14KGun** [sparcs-kaist/taxi-front#218] [Ideas]
## 디자인 되어야 할 것들
  디자인이 필요한 기능들 입니다 ㅜㅜ 아마 기획이 덜되어 수정 사항이 생길 수 있어 기획 후, 디자인이 되어야 하지 않을까 싶습니다.
  * 검색 결과 정렬
  * 신고 시스템
    * 마이페이지에 신고 기록이 추가되면 좋을 것 같습니다.
  * 영어 지원
    * 마이페이지에 언어 전환 버튼이 추가되면 좋을 것 같습니다.
  * 앱 알림
    * 마이페이지에 앱 알림 on/off가 추가 되면 좋을 것 같습니다.
  
  디자인이 덜 되었지만 일단 개발이 완료된 페이지들 입니다. 리디자인 후 리펙토링 되었으면 좋겠습니다.
  * 에러페이지
    * 뒤로가기, 홈으로 가기 버튼이 있으면 좋을 것 같습니다.
    * 에러 메시지가 함께 띄어지면 좋을 것 같습니다.
    * 아마 global state로 관리가 될 것 같아요
  * 앱 Splash 화면
 
@imYourChoi

**Comment by imYourChoi** [comment]
- 마이페이지에 들어갈 항목이 모두 결정된 후에 디자인해야 될 것 같습니다. 다른 discussion 보니 신고 내역도 필요하고 다크모드도 있어 지금 디자인에서 약 4개가 (신고내역, 영어, 앱알림, 다크모드) 추가될것 같아 모두 결정된 후에 마이페이지는 다시 손봐야 할 것 같습니다.
- 앱 splash 화면은 앱에만 사용되는 건가요? 기존의 로그인 페이지와 상당히 유사할 것 같아 고민이네요.
